.github/workflows/docker.yaml

```diff
@@ -25,6 +25,6 @@ jobs:
       - uses: docker/build-push-action@v5
         with:
           context: .
-          push: true
+          push: ${{ github.event_name != 'pull_request' || ! github.event.pull_request.head.repo.fork }}
```

Comment:
- https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
- https://docs.github.com/ja/webhooks/webhook-events-and-payloads#pull_request